src/components/bottomSheet/bottom-sheet.js

```diff
@@ -78,6 +78,35 @@ function MdBottomSheetDirective($mdBottomSheet) {
  *
  * });
  * </hljs>
+ *
+ * ### Custom Presets
+ * Developers are also able to create their own preset, which can be easily used without repeating
+ * their options each time.
+ *
+ * <hljs lang="js">
+ *   $mdBottomSheetProvider.addPreset('testPreset', {
+ *     options: function() {
+ *       return {
+ *         template:
+ *           '<md-bottom-sheet>' +
+ *             'This is a custom preset' +
+ *           '</md-bottom-sheet>',
+ *         controllerAs: 'bottomSheet',
+ *         bindToController: true,
+ *         clickOutsideToClose: true,
+ *         escapeToClose: true
+ *       };
+ *     }
+ *   });
+ * </hljs>
+ *
+ * After you created your preset at config phase, you can easily access it.
```

Comment:
Let's change `created`  to `create`

Reply:
Fixed

Reply:
Looks like you addressed the `created` -> `create` change, but I updated my comment with some more tweaks to make the sentence more clear. Please take another look at my comment above.

Reply:
Sorry, I missed it. It should be fixed now.